.rhcicd/clowdapp-backend.yaml

```diff
@@ -121,12 +121,10 @@ objects:
               name: notifications-ephemeral-data
               key: ephemeral_data.json
               optional: true
-        - name: QUARKUS_REST_CLIENT_RBAC_AUTHENTICATION_READ_TIMEOUT
+        - name: RBAC_AUTHENTICATION_MP_REST_READTIMEOUT
```

Comment:
This is probably not intented - I recall we had to hadd QUARKUS_* ?

Reply:
Yeah `*MP_REST*` is no longer working.

Reply:
Let me check. I did not want to add this.. I didn't even want to change this..